### PR TITLE
Feat/fargate support

### PR DESCRIPTION
**Pull request by MichaelBriggs-Coralogix**
# Description

Added ECS-Fargate Observability integrations (Logs/Metrics/Traces)
Added existing EKS-Fargate configuration files for linking in documentation.

# How Has This Been Tested?

Deployed each component separately via the documented steps in the README.md files. Deployed using the example cloudformation templates in the cloudformation example repository.

# Checklist:
- [ ] I have updated the relevant Helm chart(s) version(s)
- [x] I have updated the relevant component changelog(s)
- [ ] This change does not affect any particular component (e.g. it's CI or docs change)

### CODE REVIEW

**Comment by MichaelBriggs-Coralogix** [comment]
Check Documentation formatting and links (pull_request) will fail until PR for the cloudformation repository has been merged, and vice versa. Both repositories have a link to each other.

**Comment by MichaelBriggs-Coralogix** [comment]
No Helm Chart updates necessary as this is ECS.

**Review by royfur — APPROVED** [review]

File: otel-agent/eks-fargate/cx-eks-fargate-otel.yaml [hunk 1]
```diff
@@ -0,0 +1,282 @@
...
+
+    # Download kubectl
+    # If the latest stable version is not compatible with your cluster, adjust accordingly.
+    curl -LO "https://dl.k8s.io/release/$(curl -L -s https://dl.k8s.io/release/stable.txt)/bin/linux/amd64/kubectl"
+
+    # Give execute permissions to kubectl
+    chmod +x kubectl
+
+    # Run the command to label nodes
+    ./kubectl label nodes $K8S_NODE_NAME OTEL-collector-node=true
```
[the remark this answers is not in the document]
Comment (MichaelBriggs-Coralogix) [reply otel-agent/eks-fargate/cx-eks-fargate-otel.yaml:65]: Yeah, that's certainly possible. In that case, I suppose we'd want to have the main collector not collect any pods with OTEL-collector-node=true.

File: otel-agent/eks-fargate/README.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
+# OpenTelemetry EKS Fargate
```
[the remark this answers is not in the document]
Comment (MichaelBriggs-Coralogix) [reply otel-agent/eks-fargate/README.md]: Moved to new folder, otel-eks-fargate

File: otel-agent/eks-fargate/cx-eks-fargate-otel.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (MichaelBriggs-Coralogix) [reply otel-agent/eks-fargate/cx-eks-fargate-otel.yaml]: Actually, no. Just remembered that each time an EKS Fargate workload is launched it spins up on a new Fargate node. As long as they don't put additional containers into this statefulset, it'll shut down the old node when it's removed.

File: otel-agent/ecs-fargate/README.md [hunk 3]
```diff
@@ -0,0 +1,111 @@
+# ADOT (OTEL) ECS Fargate container
+
+The OpenTelemetry collector offers a vendor-agnostic implementation of how to receive, process and export telemetry data.
+
+In this document, we'll explain how to add the OTEL collector as a sidecar agent to your ECS Task Definitions.  We use an AWS customized OpenTelemetry image called AWS Distro for OpenTelemetry (ADOT), as it has several features that allow for more convenient management of the configuration. We also have an example cloudformation template for review [here](https://github.com/coralogix/cloudformation-coralogix-aws/tree/master/aws-integrations/ecs-fargate)
```
[the remark this answers is not in the document]
Comment (MichaelBriggs-Coralogix) [reply otel-agent/ecs-fargate/README.md]: Ahhh, yes, sorted! Thanks.

**Pull request merged**